README.md

```diff
@@ -75,7 +75,7 @@ Before starting the root key ceremony, the community should:
 * Identify the targets to sign and update the `targets/` directory (these may include Fulcio's CA certificate, the rekor transparency log key, the CTFE key, and SigStore's artifact signing key)
 * Identify the online keys for snapshot and timestamp roles. The key references should be updated in `scripts/step-1.5.sh`.
 
-If you are a keyholder or ceremony conductor, follow instructions [KEYHOLDER.md](KEYHOLDER.md).
+If you are a keyholder or ceremony conductor, follow instructions [KEYHOLDER.md](keyholders/OVERVIEW.md).
```

Comment:
I think you want playbooks/keyholders/OVERVIEW.md?

Reply:
... I feel silly for not trying the link. Yes!